sematic/ui/packages/common/src/pages/RunSearch/filters/common.ts

```diff
@@ -0,0 +1,3 @@
+export interface ResettableHanlde {
```

Comment:
```suggestion
export interface ResetTableHandle {
```

Reply:
After taking a second look, this is actually the word `Resettable` , which means something that can be reset.

Reply:
Also `Hanlde` -> `Handle`.